test/pkg/environment/aws/expectations.go

```diff
@@ -643,6 +646,9 @@ func (env *Environment) EventuallyExpectRoleCreated(roleName string) {
 	}).WithTimeout(30 * time.Second).WithPolling(5 * time.Second).Should(Succeed())
 }
 
+// Deletes a role and cleans up associated resources. This includes removing the EKS access entry that authorizes nodes
+// to join the cluster, detaching standard node policies (container registry, CNI, worker node, and SSM policies), and
+// finally deleting the IAM role itself.
 func (env *Environment) ExpectRoleDeleted(roleName string) {
```

Comment:
Can we do the same naming change here since this is specific to a node role with an access entry?